### PR TITLE
Add Paolo's talk to Svelte Summit

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Add talk to event description

### CODE REVIEW

File: src/calendar/2023-11-11-svelte-summit.md [hunk 1]
```diff
@@ -8,4 +8,6 @@ color: aqua
 ---
 
 Svelte Summit is an online conference dedicated to Svelte and everything that is
-happening in the community. We are supporting the event as a Gold Sponsor.
+happening in the community. Mainmatter's Senior Frontend Engineer Paolo Ricciuti
```
Comment (marcoow) [inline src/calendar/2023-11-11-svelte-summit.md]:
```suggestion
happening in the community. Mainmatter's Svelte expert Paolo Ricciuti
```

File: src/calendar/2023-11-11-svelte-summit.md [hunk 2]
```diff
@@ -8,4 +8,6 @@ color: aqua
...
+will talk about progressive enhancements in SvelteKit in his talk "Enhance!!!"
+and we are supporting the event as a Gold Sponsor.
```
Comment (marcoow) [inline src/calendar/2023-11-11-svelte-summit.md]:
```suggestion
and we are supporting the event as a Gold Sponsor as well.
```

**Review by marcoow — CHANGES_REQUESTED** [review]

File: src/workshops/telemetry-for-rust-apis.md [hunk 3]
```diff
@@ -25,7 +25,8 @@ topics:
     text: >
       We will cover Rust’s <code>Error</code> trait, with a focus on the
       information that can be retrieved and recorded in your logs; we will also
-      spend some time on logging patterns (e.g. when should an error be logged?).
+      spend some time on logging patterns (e.g. when should an error be
+      logged?).
```
Comment (marcoow) [inline src/workshops/telemetry-for-rust-apis.md:29]: this should not be in this PR

Comment (emmasofiah2o) [reply src/workshops/telemetry-for-rust-apis.md:29]: Yes. There are problems with linting. I tried rebasing but it didn't work

**Review by marcoow — APPROVED** [review]

**Pull request merged**